en/firefox/privacy/book.ftl

```diff
@@ -0,0 +1,208 @@
+
+privacy-book-the-easiest-way =  The easiest way to protect your privacy in all browsers is to stay below the radar of data collectors. If you’re using private/incognito mode, your browser won’t save a history of your activity on your device or cookies set during that browsing session.
+
+privacy-book-we-love-social =  <strong>We love social media!</strong> However, these websites (and others) might not work properly when you’re browsing privately: They will not only track what you do while being logged in, but also your activities beyond the respective network that takes place in the same browser window. For them, your browsing information is particularly valuable because they’re able to connect it to your account and might ultimately get to know you better than some of your best friends. Worried? No need to, { -brand-name-firefox } is here to help: Get the { -brand-name-facebook-container } extension, which makes it harder for { -brand-name-facebook } to track you on the web outside of { -brand-name-facebook }.
+
+privacy-book-theres-actually-even =  <strong>There’s actually even more to keep in mind when it comes to social networks:</strong> as you know, they’re made to share personal stuff with people we know (some better, some less). It’s on you to decide how much private information you want to share though. That applies to your posts as well as your location, things you like, your phone number or who you spend time with. Remember: all of this data is not only visible to the respective social network, but might also be for advertisers and the general public if you keep your profile publicly  available. Also, if you share less, there’s less that could fall into the hands of people you don’t know or trust.
+
+privacy-book-what-feels-right =  <strong>What feels right for you today might change tomorrow.</strong> And that's totally fine. It doesn't hurt to reevaluate your view on the analogue world and the web occasionally and adjust them if necessary.
+
+privacy-book-what-does-the = What does the world know about you?
+
+privacy-book-to-an-extent = To an extent, we can control what information about ourselves is available on the web. Unfortunately, this can’t protect us entirely against dangers. A particularly bad one is identity theft. Here’s what you might want to consider:
+
+privacy-book-search-your-name = <strong>Search your name online occasionally.</strong> This will give you an idea of what others find out if they’re interested in you, what information is already around, and what you might want to get removed.
+
+privacy-book-dont-have-time = <strong>Don’t have time?</strong> Set up a { -brand-name-google } Alert for your name and get notified immediately if something new pops up online.
```

Comment:
[Feedback from Peiying:](https://github.com/mozilla-l10n/www-l10n/pull/127#discussion_r523250209)
> if something new pops up online - what is this referring to? Other than breach alert, what kind of notification that is privacy related that would trigger popups?

Reply:
This is not referring to pupups but if something new comes up, anything that is indexed by Google and includes your name. 

<strong>Don’t have time?</strong> Set up a { -brand-name-google } Alert for your name and get notified if you get mentioned on a website.

Reply:
Thanks for your help, Raphael!